content/doc/upgrade-guide/2.7.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: simplepage
+title:  Jenkins Upgrade Guide
+section: doc
+---
+
+== Upgrading to Jenkins LTS 2.7.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.7.1 covers the upgrade from 1.651.3.
+
+=== Upgrading to Jenkins LTS 2.7.4
+
+This release only fixed a regression introduced in 2.7.3. No upgrade notes for this release exist.
+
+=== Upgrading to Jenkins LTS 2.7.3
```

Comment:
I would explicitly mention that 2.7.4 is recommended instead due to the serious regression impacting Windows and NFS users